app/code/Magento/QuoteBundleOptions/README.md

```diff
@@ -1,3 +1,15 @@
-# QuoteBundleOptions
+# Magento_QuoteBundleOptions module
 
-**QuoteBundleOptions** provides data provider for creating buy request for bundle products.
+This module provides data provider for creating buy request for bundle products.
+
+## Installation
+
+This module does not introduce any database schema modifications or new data.
+
+For information about a module installation in Magento 2, see [Enable or disable modules](https://devdocs.magento.com/guides/v2.4/install-gde/install/cli/install-cli-subcommands-enable.html).
+
+## Extensibility
+
+Extension developers can interact with the Magento_QuoteBundleOptions module. For more information about the Magento extension mechanism, see [Magento plug-ins](https://devdocs.magento.com/guides/v2.4/extension-dev-guide/plugins.html).
```

Comment:
Question - on DevDocs & Knowledge Base use of "Magento" is being switched to "Adobe Commerce" - should we not be doing this on these new README.md now, rather than later?

Reply:
I think we should leave a Magento as in the Readme template https://github.com/magento/devdocs/wiki/Magento-module-README.md provided by Magento.

Reply:
Accepted!